### PR TITLE
pulling early return from AI

### PR DESCRIPTION
**Pull request by chayim**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -6,13 +6,21 @@ defaults:
...
+            if [[ $CIRCLE_BRANCH == *docs ]]; then
+              echo "Identifies as documents PR, no testing required."
+              circleci step halt
+            fi
+      - run:
+          name: Early return if this branch should ignore CI
           command: |
-            if [[ $CIRCLE_BRANCH == *_docs ]]; then
-              echo "Identifies as documents PR, no testing required"
+            if [[ $CIRCLE_BRANCH == *noci ]]; then
```
Comment (rafie) [inline .circleci/config.yml:22]: Why not `_noci`? Isn't it more nice to have `yadayada_noci` than `yadayadanoci`?

**Comment by rafie** [comment]
@chayim Looks very good. One small comment.

File: .circleci/config.yml (hunk 1 above)
Comment (chayim) [reply .circleci/config.yml:22]: I have the Python habbit of using underscores as delineating. There's no good reason other than personal style. Though, I like a delineation. Feel free to pick between underscore and hyphen.

Comment (rafie) [reply .circleci/config.yml:22]: I care for consistency. Let's use an underscore now (due to `_docs`) and decide on the global convention once the generator is implemented.

Comment (chayim) [reply .circleci/config.yml]: @rafie Restored in latest push. I brought back the underscore.

**Review by rafie — APPROVED** [review]

**Pull request merged**